docs/conceptual/cryptography/keys.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Substrate Keys
+---
+
+Substrate uses multiple sets of public/private key pairs to represent
+participants of the network.
+
+## Overview
+
+Blockchain systems involve a number of participants playing different roles in
+the system. The Substrate node uses a nominated proof of stake (NPoS) system by
+default. In this system, there are two core roles:
+
+1. Validators: Users that produce blocks.
+2. Nominators: Users that support a validator.
+
+Validators need to collect funds which back their intention to produce blocks.
+Validators can provide their own funds, and also collect funds from nominators
+who trust the validator to produce new blocks correctly. As a reward for doing
+things correctly, both validators and their respective nominators get paid.
+These funds can be used for slashing if the validator misbehaves.
+
+> **Note:** Learn more about validators and nominators in the context of the
+> Substrate's NPoS [Staking module](/rustdocs/master/srml_staking/index.html).
+
+Both validators and nominators may be working with a significant value of funds,
+so Substrate has introduced abstractions which allow safer behaviors for end
+users participating in the network.
+
+These abstractions are:
+
+* [Account Keys](#account-keys): Accounts that should hold funds.
+    * [Stash Keys](#stash-keys): Cold wallets controlled by controller keys.
+    * [Controller Keys](#controller-keys): Semi-online keys that will be in the
+      direct control of a user and used to submit extrinsics.
+* [Session Keys](#session-keys): Hot keys that be must kept online by a
+  validator to perform network operations and should not hold funds.
+
+## Account Keys
+
+Account keys represent the normal kind of accounts you might expect from other
```

Comment:
This is a bit redundant from the overview, but it is a good place to state clearly that keys != accounts.

------------------------

A key pair can represent an account and control funds, like normal accounts that you would expect from other blockchains. In the context of Substrate's Balances module [link], these accounts must have a minimum amount (an "existential deposit") to exist in storage.

Account keys are defined generically and made concrete in the runtime.

To continue with our example of Stash and Controller accounts, the keys to these accounts are distinguished by their intended use, not by any underlying cryptographic difference. When creating Stash or Controller keys, all cryptography supported for normal account keys are an option.